source/plugin/api-versions.rst

```diff
@@ -7,8 +7,8 @@ This page explains which API versions exist, and to which Minecraft version thei
 +-------------+--------------+----------------+-------------------------------------------+
 | API-Version | Release Date | End of Support | Known Implementations (Minecraft Version) |
 +=============+==============+================+===========================================+
-| *8.0.0*     | TBA          | TBA            | * *SpongeForge (1.13.x)*                  |
-|             |              |                | * *SpongeVanilla (1.13.x)*                |
+| *8.0.0*     | TBA          | TBA            | * *SpongeForge*                           |
+|             |              |                | * *SpongeVanilla*                         |
 +-------------+--------------+----------------+-------------------------------------------+
```

Comment:
Is leaving the Minecraft version off intentional? Gabizou said to change it to 1.14, but I can agree with leaving it off for now given the amount of change and uncertainty for which Minecraft version will have fully-functional implementations.

Reply:
Six of one, half a dozen of the other... Leaving the MC version off an as-yet unreleased API is probably OK, as it will definitely have one by release.

Reply:
> Is leaving the Minecraft version off intentional? 
Yeah, this is the docs for api 7. No need to worry about future versions of the api.